iriscasttools/iriscasttools/__main__.py

```diff
@@ -0,0 +1,5 @@
+import sys
+from iriscasttools.main import get_iriscast_stats, parse_args
+
+cmd_args = parse_args(sys.argv[1:])
+print(get_iriscast_stats(cmd_args.as_csv, cmd_args.include_header))
```

Comment:
It would be nice to pre-configure the logger for the module like this:
https://github.com/stfc/SCD-OpenStack-Utils/blob/master/OpenStack-Rabbit-Consumer/entrypoint.py#L9

That way we can easily do something like
`LOG_LEVEL=DEBUG python3 -m iriscasttools` if something isn't working
...